docs/modules/knowledge-graph/pages/index.adoc

```diff
@@ -43,7 +43,7 @@ For more information, see xref:knowledge-graph.adoc[].
 
 == The `ragstack-ai-knowledge-store` library
 
-The `ragstack-ai-knowledge-store` library contains functions for creating a hybrid vector-and-graph knowledge store. This store combines the benefits of vector stores with the context and relationships of a related edges.
+The `ragstack-ai-knowledge-store` library contains functions for creating a hybrid vector-and-graph store. This store combines the benefits of vector stores with the context and relationships of a related edges.
```

Comment:
The `ragstack-ai-knowledge-store` library has been split, with the more convenient user facing functionality moving to `ragstack-ai-langchain`. Now, the `ragtsack-ai-knowledge-stroe` library is more of the "implementation details" (similar to cassio). Not sure we need to mention it (and we should probably mention `ragstack-ai-langchain`, etc.).

Reply:
You can still use ragstack-ai-knowledge-store if you're not using LangChain.